eng/common/docgeneration/Generate-DocIndex.ps1

```diff
@@ -65,9 +65,12 @@ function Get-TocMapping {
     $orderServiceMapping = @{}
 
     foreach ($artifact in $artifacts) {
-        $packageInfo = $metadata | ? { $_.Package -eq $artifact -and $_.Hide -ne "true" }
+        $packageInfo = $metadata | ? { $_.Package -eq $artifact }
```

Comment:
If you move this hide filter you will start getting duplicate entries by just searching for the package name. As we have some duplicate entries based on track 1 and track 2 packages in JS/Python.